src/ir_Gree.cpp

```diff
@@ -373,8 +376,8 @@ std::string IRGreeAC::toString() {
     result += F("Auto");
   else
     result += F("Manual");
-  result +=
-      String(F(", Swing Vertical Pos: ")) + uint64ToString(getSwingVerticalPosition());
+	result += F(", Swing Vertical Pos: ");
```

Comment:
Nitpick: I think the indentation is wrong here (or at least github shows it as wrong)

Reply:
Fixed.